<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGdalCredentialOptionsWidgetBase</class>
 <widget class="QWidget" name="QgsGdalCredentialOptionsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credentials</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QTableView" name="mTableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mLabelWarning">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mLabelInfo">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
